test/storage/file.js

```diff
@@ -1516,10 +1515,11 @@ describe('File', function() {
         expires: Date.now() + 5
       }, function(err) {
         var errorMessage = [
-          'Signing failed. Could not find a `private_key`.',
+          'Could not find a `private_key`.',
           'Please verify you are authorized with this property available.'
         ].join(' ');
 
+        assert.strictEqual(err.name, 'SigningError');
```

Comment:
I chose not to only because that would mean we would have to export the method, likely as a static member of `File`. I don't like when ease of testing gets in the way of sane code architecture. Sometimes, it's unavoidable, or can be worked around, but in this case, I think it's more effort than its worth.

Perhaps in the future, if we find ourselves needing to create more custom errors, we can put all the errors into their own file(s), and then testing will be easier.